doc/devel/min_dep_policy.rst

```diff
@@ -9,18 +9,28 @@ SemVer (major, minor, patch) and includes both major and minor
 releases. For projects that use date-based versioning, every release
 is a 'minor version'.
 
+Matplotlib follows `NEP 29
+<https://numpy.org/neps/nep-0029-deprecation_policy.html>`__.
+
 
 Python and NumPy
 ================
 
-- support minor versions of ``Python`` initially released
-  36 months prior to our planned release date.
-- support minor versions of ``numpy`` initially released in the 36
-  months prior to our planned release date or oldest that supports the
-  minimum python version (which ever is higher)
 
-We will bump the minimum python and numpy versions as we can every
-minor and major release, but never on a patch release.
+
```

Comment:
I would add
~~~
Matplotlib supports:
~~~
these ellipses are are bit unclear to read.

Reply:
Done, but comment is not auto-folding.